### PR TITLE
Support OpenWRT Chaos Calmer 15.05 + Hostapd

### PR DESCRIPTION
**Pull request by lzammit**
# Description

Support OpenWRT Chaos Calmer 15.05 with Hostapd
# Impacts

Switches flow
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Support for OpenWrt 15.05 with hostapd.

### CODE REVIEW

**Comment by extrafu** [comment]
I think we can safely drop support for 14.07 and only focus on 15.05.

**Comment by tlaurion** [comment]
Hostapd currently cannot share radius_das_port=3799 between ssids.

File: docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        option acct_secret 's3cr3t'
+        option dae_port '3799'
+        option dae_client '192.168.1.10'
+        option dae_secret 's3cr3t'
+        option nasid 'Lobby'
+        option encryption 'wpa2'
+        option ssid 'OpenWRT-Secure'
+----
+
+In order to apply this configuration, when you are connected using SSH on the AP, run the command 'wifi'. It will reload the configuration and broadcast the SSID. If you want to bug, you can use the command 'logread'.
```
Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc]: Me doesn't want to bug, debug is cool tho

File: docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+=================================================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+:toc: left
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
```
Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc]: Something is wrong with your formatting in asciidoc please verify, the whole guide appear 'under quotes'

File: docs/PacketFence_OpenWrt-Hostapd-14-07_Quick_Install_Guide-docinfo.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<copyright>
```
Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-14-07_Quick_Install_Guide-docinfo.xml]: This file should not be here anymore

File: docs/PacketFence_OpenWrt-Hostapd-14-07_Quick_Install_Guide.asciidoc [hunk 4]
```diff
@@ -0,0 +1,155 @@
+OpenWrt BarrierBreaker 14.07 with hostapd Quick Integration Guide
```
Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-14-07_Quick_Install_Guide.asciidoc]: This file should not be here anymore

File: docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+Go to *System -> Software*
+
+First update the repos by clicking the button Update lists if it's not up to date.
+
+Then you will have to install the packages of Hostapd and wpad.
+
+Go to the tab 'Available packages' and then search for the package hostapd into the 'Filter:' field.
+
+Click Install the hostapd package, the actual version is 2015-03-25-1.
```
Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc]: hostapd-common

File: docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+RADIUS:
+[options="compact"]
+* *Secret Passphrase*: s3cr3t
+
+Save this configuration by clicking the 'Save' button.
+
+Step 6: Troubleshoot
+~~~~~~~~~~~~~~~~~~~~
+
+Here few things you can do/check to see if you configuration is working.
```
Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc]: _your_ configuration

Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc]: Here _are_ few things

File: docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using OpenWrt CHAOS CALMER 15.05 with Hostapd.
+
+Introduction
+------------
+This guide will provide an example for the configuration of an open SSID (not encrypted) and a secured SSID (802.1X). You will need to install wpad and hostapd. These two SSIDs will do RADIUS authentication against PacketFence. You can not have both SSID configured on the same access point at the same time, there is a limitation with the DAE server.
```
Comment (whitx) [inline docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc]: _secure_ SSID

**Pull request merged**